packages/mariadb/test/env.coffee

```diff
@@ -0,0 +1,97 @@
+
+path = require 'path'
+nikita = require 'nikita'
+{env, tags} = require './test'
+{cluster} = require '../env'
+
+
+describe 'Test environment', ->
+
+  @timeout 1500000 # Extended timeout for these lengthy test
+
+  describe 'Cluster status', ->
+
+    it 'Up and running', ->
+      nikita ->
+        await @cluster.clean()
+        await @cluster.create()
+        {config} = await @lxc.state
+          $header: 'Nikita'
+          container: 'nikita'
+        config.status.should.eql 'Running'
+        {config} = await @lxc.state
+          $header: 'Node 1'
+          container: 'node1'
+        config.status.should.eql 'Running'
+        {config} = await @lxc.state
+          $header: 'Node 2'
+          container: 'node2'
+        config.status.should.eql 'Running'
+        await @cluster.clean()
+
+
+  describe 'Cluster network', ->
+
+    it 'Private IP', ->
+      nikita ->
+        await @cluster.clean()
+        await @cluster.create()
+        {$status} = await @execute
+          command: """
+          ping -c 1 #{env.nic.node1.eth1['ipv4.address']}
+          """
+        $status.should.be.true()
+        {$status} = await @execute
+          command: """
+          ping -c 1 #{env.nic.node2.eth1['ipv4.address']}
+          """
+        $status.should.be.true()
+        await @cluster.clean()
+
+  return unless tags.ssl
+
+  describe 'SSL', ->
+
+    it 'Local and remote certificates', ->
```

Comment:
It is not necessary to test the presence of the SSL/TLS releted files. Instead, we want to test that the client connection to a MariaDB is secured over SSL/TLS.